manim/scene/three_d_scene.py

```diff
@@ -146,12 +163,12 @@ def stop_3dillusion_camera_rotation(self):
 
     def move_camera(
         self,
-        phi=None,
-        theta=None,
-        distance=None,
-        gamma=None,
-        frame_center=None,
-        added_anims=[],
+        phi: Optional[float] = None,
+        theta: Optional[float] = None,
+        gamma: Optional[float] = None,
+        distance: Optional[float] = None,
+        frame_center: Optional[Union["Mobject", Sequence[float]]] = None,
+        added_anims: List["Animation"] = [],
```

Comment:
**(suggestion, typing)** Doesn't actually exclusively need to be a list. `Iterable` would be more appropriate. 
```suggestion
        added_anims: Iterable["Animation"] = [],
```